src/Components/.vscode/launch.json

```diff
@@ -1,6 +1,12 @@
 {
   "version": "0.2.0",
   "configurations": [
+    {
+      "name": ".NET Core Attach",
+      "type": "coreclr",
+      "request": "attach",
+      "processId": "${command:pickProcess}"
+    },
```

Comment:
Was this intentional?

Reply:
Yeah, this is helpful if you want to attach to debug tests in the src/Components area.